exporter.go

```diff
@@ -233,9 +233,16 @@ func main() {
 		scrapeURI = &newScrapeURI
 	}
 
+	userAgent := fmt.Sprintf("NGINX-Prometheus-Exporter/v%v", version)
+	userAgentRT := &userAgentRoundTripper{
+		agent: userAgent,
+		rt: &http.Transport{
```

Comment:
there is bug here. we need to use the `transport` created in line 220 instead of creating a new transport.